platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/more_about_tasks.adoc

```diff
@@ -275,158 +246,181 @@ Task ordering can be useful in a number of scenarios:
 ====
-.Output of **`gradle -q taskY`**
 ----
-> gradle -q taskY
+$ gradle -q taskY
 include::{snippetsPath}/tasks/mustRunAfter/tests/mustRunAfterSingleTask.out[]
 ----
 ====
 
 To specify a “must run after” or “should run after” ordering between 2 tasks, you use the link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:mustRunAfter(java.lang.Object++[]++)[Task.mustRunAfter(java.lang.Object++...++)] and link:{javadocPath}/org/gradle/api/Task.html#shouldRunAfter-java.lang.Object++...++-[Task.shouldRunAfter(java.lang.Object++...++)] methods.
-These methods accept a task instance, a task name or any other input accepted by link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:dependsOn(java.lang.Object++[]++)[Task.dependsOn(java.lang.Object++...++)].
+These methods accept a task instance, a task name, or any other input accepted by link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:dependsOn(java.lang.Object++[]++)[Task.dependsOn(java.lang.Object++...++)].
 
 Note that “`B.mustRunAfter(A)`” or “`B.shouldRunAfter(A)`” does not imply any execution dependency between the tasks:
 
 * It is possible to execute tasks `A` and `B` independently. The ordering rule only has an effect when both tasks are scheduled for execution.
 * When run with `--continue`, it is possible for `B` to execute in the event that `A` fails.
 
-As mentioned before, the “should run after” ordering rule will be ignored if it introduces an ordering cycle:
+The “should run after” ordering rule will be ignored if it introduces an ordering cycle:
 
-.A 'should run after' task ordering is ignored if it introduces an ordering cycle
 ====
 include::sample[dir="snippets/tasks/shouldRunAfterWithCycle/kotlin",files="build.gradle.kts[]"]
 include::sample[dir="snippets/tasks/shouldRunAfterWithCycle/groovy",files="build.gradle[]"]
 
-.Output of **`gradle -q taskX`**
 ----
-> gradle -q taskX
+$ gradle -q taskX
 include::{snippetsPath}/tasks/shouldRunAfterWithCycle/tests/shouldRunAfterWithCycle.out[]
 ----
 ====
 
 [[sec:adding_a_description_to_a_task]]
-== Adding a description to a task
+== Adding a description or group
 
-You can add a description to your task. This description is displayed when executing `gradle tasks`.
+You can add a description to your task and assign it to a group:
 
-.Adding a description to a task
 ====
 include::sample[dir="snippets/tasks/defineAndConfigure/kotlin",files="build.gradle.kts[tags=all]"]
 include::sample[dir="snippets/tasks/defineAndConfigure/groovy",files="build.gradle[tags=all]"]
 ====
 
+This name and description is displayed under its assigned group when executing `gradle tasks`:
+
+[source]
+----
+$ ./gradlew tasks
+
+> Task :tasks
+
+------------------------------------------------------------
+Tasks runnable from root project 'lab'
+------------------------------------------------------------
+
+Help tasks
+----------
+copy - Copies the resource directory to the target directory.
```

Comment:
❌ I find it confusing to have the `copy` task in the `help` group. Even more since we did not set a group in the snippet above.

Reply:
Updated